src/components/custom/_signup-public.js

```diff
@@ -212,11 +220,12 @@ const SignupPublic = ({
     handleSocialSignup,
     is_submitting,
 }) => {
+    const data = useStaticQuery(query)
     return (
         <Wrapper>
             <div style={{ position: 'absolute', left: '50%', height: '100%' }}>
                 <ImageWrapper ai="center">
-                    <Image img_name="deriv-platform-banner.png" width="100%" />
+                    <QueryImage data={data['deriv_platform']} alt="deriv platform" width="100%" />
```

Comment:
If it is a static string, you can use the dot notation:
```suggestion
                    <QueryImage data={data.deriv_platform} alt="deriv platform" width="100%" />
```